### DISCUSSION TITLE
Outreach to big open source projects?

### DISCUSSION THREAD

**Discussion by blaisep** [pypyr/pypyr#341] [Ideas]
The Fedora project is many things:

- A vast multi-platform infrastructure project
- A disciplined developer community
- An opportunity for newcomers to get hands-on experience with professional development
- ... maybe an opportunity to introduce pypyr as a way to make pipelines more transparent and portable?

https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/C3S3RYNWKPSDO3PCKYTZYDTUGYRBUG3F/

**Comment by yaythomas** [comment]
great idea! I don't have the bandwidth to do justice to this myself, though :-(
